projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts

```diff
@@ -45,7 +48,9 @@ const moduleComponents = [
     I18nModule,
     FeaturesConfigModule,
     PromotionsModule,
+    OrderOverviewModule,
     UrlModule,
+    SpinnerModule,
```

Comment:
Did I miss it, where did you add a spinner?

Reply:
nope you didn't miss it, SpinnerModule was already there, but I just moved it under the same import `/shared/index`

Reply:
it was in line 63 before

below the RouterModule, but I stacked it together with the other modules above for viewing consistency